master/custom/auth.py

```diff
@@ -22,6 +22,7 @@ def set_up_authorization(settings):
                 util.StopBuildEndpointMatcher(
                     role="buildbot-owners", defaultDeny=False
                 ),
+                util.StopBuildEndpointMatcher(role="python-triage"),
```

Comment:
I *think* you need `defaultDeny=False` here.  I'm not certain of that, it's been quite a while since I read the docs for this.

While we're in here, we might as well allow triagers to rebuild or force, too.

Reply:
Thanks for the review. I have addressed it :)